spec/features/course/unread_status_management_spec.rb

```diff
@@ -39,7 +40,8 @@
     end
 
     describe 'sidebar' do
-      let!(:announcement) { create(:course_announcement, course: course, creator: second_user) }
+      let!(:announcement) { create(:course_announcement, course: course, creator: second_user,
+                                   updater: first_user) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 44, 57 should be on its own line.